src/coalesce.rs

```diff
@@ -62,9 +64,17 @@ impl Serialize for Event {
     fn serialize<S>(&self, s: S) -> Result<S::Ok, S::Error>
     where S: Serializer
     {
-        let mut map = s.serialize_map(Some(1 + self.body.values.len()))?;
+        let mut length = 1 + self.body.values.len();
+        if let Some(_) = self.node {
+            length += 1
+        };
+        let mut map = s.serialize_map(Some(length))?;
         map.serialize_key("ID")?;
         map.serialize_value(&self.id)?;
+        if let Some(node) = &self.node {
+            map.serialize_key("NDOE")?;
```

Comment:
Small typo here, works well otherwise! Thank you very much :+1:

Reply:
oops. ;-) Will fix.